src/lib/utils/deepLinking.web.js

```diff
@@ -17,6 +17,11 @@ class DeepLinkingWeb {
 
     return params
   }
+
+  clearQuery() {
+    const urlWithoutQuery = this.link.split('?')[0]
```

Comment:
use URL class https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

Reply:
Based on slack comment by @sirpy 
I applied an alternative solution: 'Handle in UI and do not attempt to connect a previously used incoming URI'

See last commit